src/main/java/racinggame/domain/Car.java

```diff
@@ -0,0 +1,73 @@
+package racinggame.domain;
+
+import static racinggame.view.OutputView.*;
+
+import java.util.Objects;
+
+public class Car {
+
+    public static final int MIN_MOVE_NUMBER = 0;
+    public static final int MAX_MOVE_NUMBER = 9;
+
+    private CarPosition position;
+    private final CarName name;
+
+    public Car(String name, CarPosition position) {
+        this.name = new CarName(name);
+        this.position = position;
+    }
+
```

Comment:
```suggestion
    public Car(CarName name, CarPosition position) {
        this.name = name;
        this.position = position;
    }
```
주 생성자는 필드와 완전히 일치하는 타입의 생성자를 주 생성자라 하고 주 생성자는 필수이다.